dataactvalidator/config/sqlrules/d1_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,8 @@
+SELECT
+    row_number,
+    record_type,
+    fain
+FROM detached_award_financial_assistance as dafa
+WHERE dafa.submission_id = {0}
+AND dafa.record_type = 2
```

Comment:
We should probably indent the lines with `AND` here to keep formatting consistent and easy to view